demos/CMakeLists.txt

```diff
@@ -37,7 +37,7 @@ macro(CREATE_DEMO _target _sources)
     target_link_libraries(${_target}
         freetype-gl
         ${OPENGL_LIBRARY}
-        ${GLFW3_LIBRARY}
+        glfw ${GLFW_LIBRARIES}
```

Comment:
Also why does this contain GLFW3_LIBRARY AND GLFW_LIBRARIES.  The tools only support GLFW3.